src/blocks.js

```diff
@@ -3327,17 +3336,50 @@ BlockMorph.prototype.showHelp = function () {
         }
     }
 
-    pic.onload = function () {
-        help = newCanvas(new Point(pic.width, pic.height), true); // nonRetina
-        ctx = help.getContext('2d');
-        ctx.drawImage(pic, 0, 0);
-        new DialogBoxMorph().inform(
-            'Help',
-            null,
-            myself.world(),
-            help
-        );
-    };
+    if (lang === 'old') {
```

Comment:
Why do we need a separate code path for the old/not updated images? Can't we just display them in the same scroll frame? It doesn't seem to me like they work any differently, and if we can do that, then we can have just one set of images, and simplify the error handling function.

Reply:
Yeah, they could probably be combined. If I remember correctly, I kept the old popup code path because:
- The new images are double-sized and scaled down so they don't look terrible on retina displays, while the old ones aren't.
- The new images are all 572 px wide, while the old ones are variable width.
- The new images have padding to make room for a scroll bar on the right side, while the old ones don't.

The first two should be easy to solve. For the last, I'd need to remove the padding from every new image and just add it in when it's drawn in this step.

Reply:
Fixed in 283a82c